internal/nginx-meshctl/deploy/deploy_test.go

```diff
@@ -43,7 +44,41 @@ var _ = Describe("Deploy", func() {
 		// we need to set either tracing/telemetry to nil
 		values.Telemetry = nil
 		values.Tracing = nil
-		Expect(deployer.Deploy()).To(Succeed())
+		_, err := deployer.Deploy()
+		Expect(err).To(Succeed())
```

Comment:
Our typical wording here is `Expect(err).ToNot(HaveOccurred())` (same for all subsequent cases of this)